pkgs/applications/misc/xygrib/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, bzip2, qtbase, qttools, libnova, proj, libpng, openjpeg } :
+
+stdenv.mkDerivation rec {
+  version = "1.2.6.1";
+  name = "xygrib-${version}";
```

`pname` does the version interpolation on it's own.
```suggestion
  pname = "xygrib";
```